pkg/orchestrator/etcd_worker.go

```diff
@@ -120,7 +121,8 @@ func (worker *EtcdWorker) Run(ctx context.Context, session *concurrency.Session,
 		sessionDone = make(chan struct{})
 	}
 	lastReceivedEventTime := time.Now()
-
+	// limit the frequency of EtcdWorker ticks
+	rl := rate.NewLimiter(10, 30)
```

Comment:
Can you add some comments on the meaning of the numbers? And ideally they should be configurable.

Reply:
don